homeassistant/components/surepetcare/entity.py

```diff
@@ -0,0 +1,46 @@
+"""Entity for Surepetcare."""
+from __future__ import annotations
+
+from abc import abstractmethod
+
+from surepy.entities import SurepyEntity
+
+from homeassistant.core import callback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import SurePetcareDataCoordinator
+
+
+class SurePetcareEntity(CoordinatorEntity):
+    """An implementation for Sure Petcare Entities."""
+
+    def __init__(
+        self,
+        _id: int,
```

Comment:
Python convention is that underscore prefix indicates the variable will not be used. I suggest renaming this to `surepetcare_id` or something.